frontend/src/static/css/v_authorship.scss

```diff
@@ -120,6 +120,30 @@
             white-space: pre-wrap;
           }
         }
+
+        .title {
+          .clickable {
+            .fa-caret-down {
+              display: inherit;
+            }
+
+            .fa-caret-right {
+              display: none;
+            }
+          }
+        }
+      }
+
+      .title {
```

Comment:
Is it necessary to have this twice? Can you try with having this only once? If no suitable solution, we will go ahead with this implemention

Reply:
I can't think of a better solution.

But there are several instances of similar patterns in the repo, such as [here](https://github.com/reposense/RepoSense/blob/bf77c18fd41e53676458e80fd07922a215c7472c/frontend/src/static/css/v_authorship.scss#L103-L124) and [here](https://github.com/reposense/RepoSense/blob/bf77c18fd41e53676458e80fd07922a215c7472c/frontend/src/static/css/v_authorship.scss#L196-L235)

If we ever find a more elegant solution we can improve on all of them. But for now I think this is fine.

Reply:
Why not utilize the `file` (collapsed) and `file active` (expanded) classes to deal with this?

Reply:
Could you elaborate more on that? I thought that is what I am currently doing. My current implementation is to change the display based on whether the class `active` exists.

Reply:
> Could you elaborate more on that? I thought that is what I am currently doing. My current implementation is to change the display based on whether the class `active` exists.

Yes now I see it. Sry about that, couldn't see the indentation properly.

Aside from that, would it be better if you move lines 137-146 to be inside the existing `.title` class (lines 126 and beyond)?

Reply:
I guess we can split them apart, but I feel that keeping the 2 sections together makes it more immediately obvious to the reader about their behaviours.

Reply:
Leaving this open for others to make their input if necessary.